### PR TITLE
fixed problems with schemas, new step in common steps and documented …

### PR DESCRIPTION
**Pull request by nicolasvallejo13**
There is a missing step in the scenario "CREATE and DELETE a board" which will be added in another PR once the step 'the response contains an error message' is in master https://github.com/atbootcamp/bootcamp04-trello/issues/30

```
Feature: Trello Boards API # tests/api/features/boards.feature:1
  As a user I want to use Boards API and perform Get,Post And Delete operations
  Scenario: Get all information of board             # tests/api/features/boards.feature:4
    Given I am a trello user                         # tests/api/features/steps/common_steps.py:31
    When I send a "GET" request to "/boards/{board}" # tests/api/features/steps/common_steps.py:8
    Then the status code is "200"                    # tests/api/features/steps/common_steps.py:23
    And I receive a response with the "board" schema # tests/api/features/steps/common_steps.py:27

  Scenario: Get members of board                             # tests/api/features/boards.feature:10
    Given I am a trello user                                 # tests/api/features/steps/common_steps.py:31
    When I send a "GET" request to "/boards/{board}/members" # tests/api/features/steps/common_steps.py:8
    Then the status code is "200"                            # tests/api/features/steps/common_steps.py:23
    And I receive a list with at least "1" "{member}"        # tests/api/features/steps/common_steps.py:14

  Scenario: Updating information of a Board               # tests/api/features/boards.feature:16
    Given I am a trello user                              # tests/api/features/steps/common_steps.py:31
    When I send a "PUT" request to "/boards/{board}"      # tests/api/features/steps/common_steps.py:8
      | Key  | Value              |
      | name | testingnicolasv    |
      | desc | a valid data table |
    Then the status code is "200"                         # tests/api/features/steps/common_steps.py:23
    And verify the new object contains the following info # tests/api/features/steps/boards_steps.py:10
      | Key  | Value              |
      | name | testingnicolasv    |
      | desc | a valid data table |

  Scenario Outline: CREATE and DELETE a board -- @1.1        # tests/api/features/boards.feature:38
    Given the user creates a board with "name" to be "kiara" # tests/api/features/steps/boards_steps.py:23
    Then the status code is "200"                            # tests/api/features/steps/common_steps.py:23
    And I receive a response with the "board" schema         # tests/api/features/steps/common_steps.py:27
    Then the user deletes the board "kiara"                  # tests/api/features/steps/boards_steps.py:38
    And the status code is "200"                             # tests/api/features/steps/common_steps.py:23

  Scenario Outline: CREATE and DELETE a board -- @1.2       # tests/api/features/boards.feature:39
    Given the user creates a board with "name" to be "jose" # tests/api/features/steps/boards_steps.py:23
    Then the status code is "200"                           # tests/api/features/steps/common_steps.py:23
    And I receive a response with the "board" schema        # tests/api/features/steps/common_steps.py:27
    Then the user deletes the board "jose"                  # tests/api/features/steps/boards_steps.py:38
    And the status code is "200"                            # tests/api/features/steps/common_steps.py:23

  Scenario Outline: CREATE and DELETE a board -- @1.3          # tests/api/features/boards.feature:40
    Given the user creates a board with "name" to be "Nicolas" # tests/api/features/steps/boards_steps.py:23
    Then the status code is "200"                              # tests/api/features/steps/common_steps.py:23
    And I receive a response with the "board" schema           # tests/api/features/steps/common_steps.py:27
    Then the user deletes the board "Nicolas"                  # tests/api/features/steps/boards_steps.py:38
    And the status code is "200"                               # tests/api/features/steps/common_steps.py:23

  Scenario Outline: CREATE and DELETE a board -- @1.4        # tests/api/features/boards.feature:41
    Given the user creates a board with "name" to be "keila" # tests/api/features/steps/boards_steps.py:23
    Then the status code is "200"                            # tests/api/features/steps/common_steps.py:23
    And I receive a response with the "board" schema         # tests/api/features/steps/common_steps.py:27
    Then the user deletes the board "keila"                  # tests/api/features/steps/boards_steps.py:38
    And the status code is "200"                             # tests/api/features/steps/common_steps.py:23

  Scenario Outline: CREATE and DELETE a board -- @1.5          # tests/api/features/boards.feature:42
    Given the user creates a board with "name" to be "Vallejo" # tests/api/features/steps/boards_steps.py:23
    Then the status code is "200"                              # tests/api/features/steps/common_steps.py:23
    And I receive a response with the "board" schema           # tests/api/features/steps/common_steps.py:27
    Then the user deletes the board "Vallejo"                  # tests/api/features/steps/boards_steps.py:38
    And the status code is "200"                               # tests/api/features/steps/common_steps.py:23

  Scenario Outline: CREATE and DELETE a board -- @1.6        # tests/api/features/boards.feature:43
    Given the user creates a board with "name" to be "pabon" # tests/api/features/steps/boards_steps.py:23
    Then the status code is "200"                            # tests/api/features/steps/common_steps.py:23
    And I receive a response with the "board" schema         # tests/api/features/steps/common_steps.py:27
    Then the user deletes the board "pabon"                  # tests/api/features/steps/boards_steps.py:38
    And the status code is "200"                             # tests/api/features/steps/common_steps.py:23

  Scenario: GET checklist of board                              # tests/api/features/boards.feature:45
    Given I am a trello user                                    # tests/api/features/steps/common_steps.py:31
    When I send a "GET" request to "/boards/{board}/checklists" # tests/api/features/steps/common_steps.py:8
    Then the status code is "200"                               # tests/api/features/steps/common_steps.py:23
    And I receive a list with at least "1" "{checklist}"        # tests/api/features/steps/common_steps.py:14
      Assertion Failed: No {checklist}s were found
```

### CODE REVIEW

File: tests/api/features/boards.feature [hunk 1]
```diff
@@ -23,7 +26,7 @@ Feature: Trello Boards API
             | desc        | a valid data table         |
```
Comment (mefemefe) [inline tests/api/features/boards.feature:27]: This Scenario has no validations 🤔 (THEN step?)

Comment (mefemefe) [reply tests/api/features/boards.feature:27]: My bad! It looked like this:
pr
So I thought all the hidden lines were part of the same table

**Comment by nicolasvallejo13** [comment]
All recommended changes done

**Comment by nicolasvallejo13** [comment]
```
Feature: Trello Boards API # tests/api/features/boards.feature:1
  As a user I want to use Boards API and perform Get,Post And Delete operations
  @somke
  Scenario: Get all information of board             # tests/api/features/boards.feature:5
    Given A board is created                         # tests/api/features/steps/boards_steps.py:52
    When I send a "GET" request to "/boards/{board}" # tests/api/features/steps/common_steps.py:8
    Then the status code is "200"                    # tests/api/features/steps/common_steps.py:23
    And I receive a response with the "board" schema # tests/api/features/steps/common_steps.py:27

  @somke
  Scenario: Get members of board                             # tests/api/features/boards.feature:11
    Given A board is created                                 # tests/api/features/steps/boards_steps.py:52
    When I send a "GET" request to "/boards/{board}/members" # tests/api/features/steps/common_steps.py:8
    Then the status code is "200"                            # tests/api/features/steps/common_steps.py:23
    And I receive a list with at least "1" "member"          # tests/api/features/steps/common_steps.py:14

  @somke
  Scenario: Updating information of a Board               # tests/api/features/boards.feature:17
    Given A board is created                              # tests/api/features/steps/boards_steps.py:52
    When I send a "PUT" request to "/boards/{board}"      # tests/api/features/steps/common_steps.py:8
      | Key  | Value              |
      | name | testingnicolasv    |
      | desc | a valid data table |
    Then the status code is "200"                         # tests/api/features/steps/common_steps.py:23
    And verify the new object contains the following info # tests/api/features/steps/boards_steps.py:10
      | Key  | Value              |
      | name | testingnicolasv    |
      | desc | a valid data table |

  @somke
  Scenario Outline: CREATE and DELETE a board -- @1.1        # tests/api/features/boards.feature:41
    Given the user creates a board with "name" to be "kiara" # tests/api/features/steps/boards_steps.py:23
    Then the status code is "200"                            # tests/api/features/steps/common_steps.py:23
    And I receive a response with the "board" schema         # tests/api/features/steps/common_steps.py:27
    Then the user deletes the board "kiara"                  # tests/api/features/steps/boards_steps.py:38
    And verify the board does not exist                      # tests/api/features/steps/boards_steps.py:60
    Then the status code is "404"                            # tests/api/features/steps/common_steps.py:23

  @somke
  Scenario Outline: CREATE and DELETE a board -- @1.2       # tests/api/features/boards.feature:42
    Given the user creates a board with "name" to be "jose" # tests/api/features/steps/boards_steps.py:23
    Then the status code is "200"                           # tests/api/features/steps/common_steps.py:23
    And I receive a response with the "board" schema        # tests/api/features/steps/common_steps.py:27
    Then the user deletes the board "jose"                  # tests/api/features/steps/boards_steps.py:38
    And verify the board does not exist                     # tests/api/features/steps/boards_steps.py:60
    Then the status code is "404"                           # tests/api/features/steps/common_steps.py:23

  @somke
  Scenario Outline: CREATE and DELETE a board -- @1.3          # tests/api/features/boards.feature:43
    Given the user creates a board with "name" to be "Nicolas" # tests/api/features/steps/boards_steps.py:23
    Then the status code is "200"                              # tests/api/features/steps/common_steps.py:23
    And I receive a response with the "board" schema           # tests/api/features/steps/common_steps.py:27
    Then the user deletes the board "Nicolas"                  # tests/api/features/steps/boards_steps.py:38
    And verify the board does not exist                        # tests/api/features/steps/boards_steps.py:60
    Then the status code is "404"                              # tests/api/features/steps/common_steps.py:23

  @somke
  Scenario Outline: CREATE and DELETE a board -- @1.4        # tests/api/features/boards.feature:44
    Given the user creates a board with "name" to be "keila" # tests/api/features/steps/boards_steps.py:23
    Then the status code is "200"                            # tests/api/features/steps/common_steps.py:23
    And I receive a response with the "board" schema         # tests/api/features/steps/common_steps.py:27
    Then the user deletes the board "keila"                  # tests/api/features/steps/boards_steps.py:38
    And verify the board does not exist                      # tests/api/features/steps/boards_steps.py:60
    Then the status code is "404"                            # tests/api/features/steps/common_steps.py:23

  @somke
  Scenario Outline: CREATE and DELETE a board -- @1.5          # tests/api/features/boards.feature:45
    Given the user creates a board with "name" to be "Vallejo" # tests/api/features/steps/boards_steps.py:23
    Then the status code is "200"                              # tests/api/features/steps/common_steps.py:23
    And I receive a response with the "board" schema           # tests/api/features/steps/common_steps.py:27
    Then the user deletes the board "Vallejo"                  # tests/api/features/steps/boards_steps.py:38
    And verify the board does not exist                        # tests/api/features/steps/boards_steps.py:60
    Then the status code is "404"                              # tests/api/features/steps/common_steps.py:23

  @somke
  Scenario Outline: CREATE and DELETE a board -- @1.6        # tests/api/features/boards.feature:46
    Given the user creates a board with "name" to be "pabon" # tests/api/features/steps/boards_steps.py:23
    Then the status code is "200"                            # tests/api/features/steps/common_steps.py:23
    And I receive a response with the "board" schema         # tests/api/features/steps/common_steps.py:27
    Then the user deletes the board "pabon"                  # tests/api/features/steps/boards_steps.py:38
    And verify the board does not exist                      # tests/api/features/steps/boards_steps.py:60
    Then the status code is "404"                            # tests/api/features/steps/common_steps.py:23

  @negative
  Scenario: Updating information of a Board negative  # tests/api/features/boards.feature:49
    Given A board is created                          # tests/api/features/steps/boards_steps.py:52
    When I send a "PUT" request to "/boards/{board}"  # tests/api/features/steps/common_steps.py:8
      | Key         | Value      |
      | nagrsgsgsdg | testxcasv  |
      | dvcv        | a vxxtable |
    Then the status code is "200"                     # tests/api/features/steps/common_steps.py:23

  @negative
  Scenario: Get members of board negative                   # tests/api/features/boards.feature:59
    Given A board is created                                # tests/api/features/steps/boards_steps.py:52
    When I send a "GET" request to "/boards/{board}/dgzdgz" # tests/api/features/steps/common_steps.py:8
    Then the status code is "404"                           # tests/api/features/steps/common_steps.py:23

  @negative
  Scenario Outline: Get a board negative -- @1.1                       # tests/api/features/boards.feature:71
    Given I send a "GET" request to "/boards/6290f7ea8abc4849607f0a52" # tests/api/features/steps/common_steps.py:8
    Then the status code is "404"                                      # tests/api/features/steps/common_steps.py:23

  @negative
  Scenario Outline: Get a board negative -- @1.2              # tests/api/features/boards.feature:72
    Given I send a "GET" request to "/boards/6290f7ea8abc484" # tests/api/features/steps/common_steps.py:8
    Then the status code is "400"                             # tests/api/features/steps/common_steps.py:23

  @negative
  Scenario Outline: Get a board negative -- @1.3                       # tests/api/features/boards.feature:73
    Given I send a "GET" request to "/boards/628bb6a0f8621c367f942e6b" # tests/api/features/steps/common_steps.py:8
    Then the status code is "404"                                      # tests/api/features/steps/common_steps.py:23

  @negative
  Scenario Outline: Get a board negative -- @1.4              # tests/api/features/boards.feature:74
    Given I send a "GET" request to "/boards/6290f7ea8abc4H4" # tests/api/features/steps/common_steps.py:8
    Then the status code is "400"                             # tests/api/features/steps/common_steps.py:23

  @negative
  Scenario Outline: Get a board negative -- @1.5              # tests/api/features/boards.feature:75
    Given I send a "GET" request to "/boards/6290f7ea8abc4H4" # tests/api/features/steps/common_steps.py:8
    Then the status code is "400"   
```

**Review by nicolasvallejo13 — COMMENTED** [review]
fixed conflicts

**Review by nicolasvallejo13 — COMMENTED** [review]
fixign conflicts

File: tests/api/features/boards.feature [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
     As a user I want to use Boards API and perform Get,Post And Delete operations
    
+   @somke
    Scenario: Get all information of board 
+        Given A board is created
         When I send a "GET" request to "/boards/{board}"
         Then the status code is "200"
         And I receive a response with the "board" schema
-
+    @somke
```
Comment (mefemefe) [inline tests/api/features/boards.feature:10]: Typo:
```suggestion
    @smoke  
```

File: tests/api/features/boards.feature [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
-
+    @somke  
     Scenario: Get members of board 
+        Given A board is created
         When I send a "GET" request to "/boards/{board}/members"
         Then the status code is "200"
-        And verify there is at least 1 member
-    
+        And I receive a list with at least "1" "member"
+    @somke
```
Comment (mefemefe) [inline tests/api/features/boards.feature:16]: Typo:
```suggestion
    @smoke
```

File: tests/api/features/boards.feature [hunk 4]
```diff
@@ -22,15 +26,51 @@ Feature: Trello Boards API
             | name        | testingnicolasv            |
             | desc        | a valid data table         | 
 
-
-     Scenario Outline: CREATE and DELETE a board 
+    
+    @somke
```
Comment (mefemefe) [inline tests/api/features/boards.feature:30]: Typo:
```suggestion
    @smoke
```

**Review by eduardomcjl — APPROVED** [review]

**Review by mefemefe — APPROVED** [review]

**Pull request merged**